src/handlers/indexers/create_indexer.rs

```diff
@@ -37,10 +54,19 @@ async fn build_create_indexer_request(request: &mut Multipart) -> Result<CreateI
             "script.js" => {
                 create_indexer_request.data = field.bytes().await.map_err(IndexerError::FailedToReadMultipartField)?
             }
-            "webhook_url" => {
-                create_indexer_request.webhook_url =
+            "target_url" => {
+                create_indexer_request.target_url =
                     field.text().await.map_err(IndexerError::FailedToReadMultipartField)?
             }
+            "table_name" => {
+                create_indexer_request.table_name =
+                    Some(field.text().await.map_err(IndexerError::FailedToReadMultipartField)?)
+            }
```

Comment:
does that mean it fails if field not specified ?

Reply:
Not it fails if `table_name` is specified and we fail to read it